cmd/commands/wiring.go

```diff
@@ -220,5 +220,12 @@ func installKubeConfigSupport(command *cobra.Command, client *core.Client, kc *c
 		}
 		return nil
 	}
+}
 
+func FindSubcommand(command *cobra.Command, names ...string) *cobra.Command {
```

Comment:
Can this be made private ?

Reply:
It's used in `github.com/projectriff/riff/cmd/commands` and `github.com/projectriff/riff/distro/commands`

Reply:
It's used in `github.com/projectriff/riff/cmd/commands` and `github.com/projectriff/riff/distro/commands`

Reply:
It's used in `github.com/projectriff/riff/cmd/commands` and `github.com/projectriff/riff/distro/commands`

Reply:
It's used in `github.com/projectriff/riff/cmd/commands` and `github.com/projectriff/riff/distro/commands`

Reply:
Maybe move to `cobra.go` then

Reply:
Maybe move to `cobra.go` then

Reply:
👍